soap/src/java/com/zimbra/soap/account/message/GetInfoResponse.java

```diff
@@ -504,6 +512,14 @@ public String getSoapURL() {
         return Prop.toMultimap(props, userPropKey);
     }
 
+    public Boolean getIsTrackingIMAP() {
```

Comment:
Don't we want to return 'false' in the case of the value not being returned from the server?
The only side-effect would be a perhaps unneeded 'BeginTrackingImapRequest' being submitted.